salt/config/__init__.py

```diff
@@ -1409,7 +1400,6 @@ def _gather_buffer_space():
     'http_max_body': 100 * 1024 * 1024 * 1024,  # 100GB
     'python2_bin': 'python2',
     'python3_bin': 'python3',
-    'cache': 'localfs',
```

Comment:
Yeah, this was a screw-up on my part.  I noticed that the latest updates from develop did not get applied to my branch and when I PRed it appears I'm undoing what was already done.  In the process of addressing this now.